index.d.ts

```diff
@@ -1,8 +1,4 @@
-type Action = {
-  type: string;
-};
-
-type Reducer<S> = (state: S, action: Action) => S;
+import { Reducer } from 'redux';
```

Comment:
If you want to avoid this dev dependency, I can just copy the type from Redux instead.